### ISSUE TITLE
HTTP Request 

### ISSUE THREAD

**Issue by UCYT5040** [radimitrov/CSharpShellApp#179]
Hello. I am trying to make an HTTP request.
I get some error about security.
I'm not sure why it isn't working.
![Screenshot_20220721-230427](https://user-images.githubusercontent.com/46662314/180360233-cb289755-8195-4e49-a78e-2d4a5cabdee2.png)

Thank you for maintaining a great app, I hope this can be resolved.

**Comment by radimitrov** [comment]
That is an issue on the SSL side. Probably an expired certificate the server is sending. It's up to you if you want to bypass it.
https://stackoverflow.com/questions/54431583/is-this-the-correct-solve-for-certificate-verify-failed-exception

**Comment by UCYT5040** [comment]
![image](https://user-images.githubusercontent.com/46662314/180879818-435f160a-cae4-4d0b-85bd-d542bb1ddcd6.png)
The certificate is valid, here is it working on Chrome browser.

**Comment by radimitrov** [comment]
Apparently it's a Xamarin issue with Let's Encrypt.
Tomorrow I'll see if switching HttpClient handlers fixes it.
https://docs.microsoft.com/en-us/answers/questions/535932/xamarin-android-throws-error-ssl-routinesopenssl-i.html

**Comment by UCYT5040** [comment]
Hello sir.
Did switching to HttpClient handlers fix it?

**Comment by radimitrov** [comment]
No, it didn't work. The hacky workaround is the only way for now.
However, it seems to be working with a basic .NET 6 Android project.
So that could be a fix for when I convert the app to that.
